dspace-server-webapp/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -2758,17 +2761,182 @@ public void testHiddenMetadataForUserWithWriteRights() throws Exception {
+
+        getClient(token).perform(get("/api/core/items/" + item.getID()))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$", ItemMatcher.matchItemProperties(item)))
+            .andExpect(jsonPath("$.entityType", is(emptyOrNullString())));
+    }
+
+    @Test
+    public void testEntityTypeModification() throws Exception {
+        context.turnOffAuthorisationSystem();
+        EntityType publication = EntityTypeBuilder.createEntityTypeBuilder(context, "Publication").build();
+        Community community = CommunityBuilder.createCommunity(context)
+            .withName("Parent Community")
+            .build();
+        Collection collection = CollectionBuilder.createCollection(context, community).withName("Collection").build();
+        Item item = ItemBuilder.createItem(context, collection)
+            .withTitle("Publication1")
+            .withAuthor("Testy, TEst")
+            .withIssueDate("2015-01-01")
+            .build();
+        context.restoreAuthSystemState();
+
+        // Verify there is no entityType yet
+        String token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(get("/api/core/items/" + item.getID()))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$", ItemMatcher.matchItemProperties(item)))
+            .andExpect(jsonPath("$.entityType", is(emptyOrNullString())));
+
+        // Modify the entityType and verify the response already contains this modification
+        List<Operation> ops = new ArrayList<>();
+        List<Map<String, String>> values = new ArrayList<>();
+        Map<String, String> value = new HashMap<>();
+        value.put("value", "Publication");
+        values.add(value);
+        AddOperation addOperation = new AddOperation("/metadata/relationship.type", values);
+        ops.add(addOperation);
+        String patchBody = getPatchContent(ops);
+        getClient(token).perform(patch("/api/core/items/" + item.getID())
+                .content(patchBody)
+                .contentType(contentType))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$", ItemMatcher.matchItemProperties(item)))
+            .andExpect(jsonPath("$.entityType", is("Publication")));
+
+        // Verify this modification is permanent
+        getClient(token).perform(get("/api/core/items/" + item.getID()))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$", ItemMatcher.matchItemProperties(item)))
+            .andExpect(jsonPath("$.entityType", is("Publication")));
+    }
+
     @Test
     public void findOneTestWithEmbedsWithNoPageSize() throws Exception {
         context.turnOffAuthorisationSystem();
 
         parentCommunity = CommunityBuilder.createCommunity(context)
-                                          .withName("Parent Community")
-                                          .build();
+            .withName("Parent Community")
```

Comment:
Starting here, your IDE appears to have made a large number of formatting changes.  Could you revert these changes, as it would make this PR a lot smaller & easier to review.... currently over 1/2 the PR is just code realignment.